package.json

```diff
@@ -31,17 +31,17 @@
   },
   "homepage": "https://github.com/holidaycheck/react-google-tag-manager#readme",
   "devDependencies": {
-    "babel-cli": "^6.8.0",
-    "babel-preset-es2015": "6.9.0",
-    "babel-preset-react": "6.5.0",
-    "babel-register": "6.9.0",
-    "chai": "^3.3.0",
-    "chai-string": "^1.1.3",
-    "eslint": "2.12.0",
-    "eslint-config-holidaycheck": "0.9.0",
-    "eslint-plugin-react": "5.1.1",
-    "mocha": "^2.5.0",
```

Comment:
This works in the current version and I have no idea who will take over. Feel free to change it again. :)